schema/tests/Provider/DestinationRate/DestinationRateLifeCycleTest.php

```diff
@@ -35,26 +35,27 @@ protected function createDto()
      */
     protected function addDestinationRate()
     {
-        $ddiProviderRegistrationDto = $this->createDto();
+        $destinationRateDto = $this->createDto();
 
         /** @var DestinationRate $ddiProviderRegistration */
         $ddiProviderRegistration = $this->entityTools
```

Comment:
ddiProviderRegistration in variable and comment above seems also wrong

Reply:
:+1: